donkeycar/templates/train.py

```diff
@@ -177,6 +186,10 @@ def main():
     tubs = args['--tubs']
     model = args['--model']
     model_type = args['--type']
+    model_name, model_ext = os.path.splitext(model)
+    is_tflite = model_ext == '.tflite'
+    if is_tflite:
+        model = model_name + '.h5'
```

Comment:
Prefer string interpolation here. So something like `f'{model_name}.h5'`. Works on Python 3.6 +

Reply:
I'll have a look at this - I need to merge in some other small changes for RPi setup in a follow up PR.